corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -728,23 +728,33 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", [
         submitAction: function (e) {
             var self = this;
             e.preventDefault();
-            self.performSubmit();
+            self.performSubmit(formplayerConstants.requestInitiatedByTagsMapping.USER_CLICK_SUBMIT);
```

Comment:
hm the original/current use case of the `initiatedBy` is to add tags to DD (this [PR](https://github.com/dimagi/commcare-hq/pull/34006)) This use case isn't really the intended purpose and I guess this is minor but each unique DD tag does add additional cost. Maybe it's better off to make this a new argument?

Reply:
good call, no need to add tags that no one is asking for.